documentation/book/master.adoc

```diff
@@ -21,6 +21,8 @@ include::assembly-security.adoc[leveloffset=+1]
 
 include::assembly-upgrade.adoc[leveloffset=+1]
 
+include::proc-uninstalling.adoc[leveloffset=+1]
```

Comment:
As a user, wouldn't uninstallation be expected somwhere after installation?

Reply:
Personally, I think including it right at the end of the regular chapters is the right approach.

Reply:
At the end seems best to me and it's the way it's presented in AMQ Online too.